lib/api/ssr.ts

```diff
@@ -136,26 +135,3 @@ export const server =
   process.env.NODE_ENV !== "production"
     ? "http://localhost:3000"
     : "https://explorer.livepeer.org";
-
```

Comment:
I think this get removed due to it not being used. It might be used again if we decide to improve performance as discussed in the code audit. (See `lib/api/ssr`).

Reply:
You're saying the `server` variable can be removed?

Reply:
I meant to comment on `getEnsIdentity` being removed.

Reply:
Removed in 0ab6267d0110c3c48fc8e45d86e253e14ffb2994